manifest.yml

```diff
@@ -6,6 +6,6 @@ declared-services:
 applications:
 - name: sample-nodejs-cloudant
   random-route: true
-  memory: 512M
+  memory: 192M
```

Comment:
Should we do 256M here so there is more of a buffer for people who make changes to the app?